content/en/docs/concepts/security/security-checklist.md

```diff
@@ -206,6 +206,24 @@ SELinux is only available on Linux nodes, and enabled in
 [some Linux distributions](https://en.wikipedia.org/wiki/Security-Enhanced_Linux#Implementations).
 {{< /note >}}
 
+## Logs and auditing
+
+- [ ] Audit logs, if enabled, are protected from general access.
+- [ ] The `/logs` API is disabled (you are running kube-apiserver with
+  `--enable-logs-handler=false`).
+
+  Kubernetes includes a deprecated `/logs` API endpoint, enabled by default,
+  that lets users request API server logs over HTTP. Accessing that endpoint
```

Comment:
```suggestion
  that lets users request the contents of the /var/log directory over HTTP. Accessing that endpoint
```

Reply:
Is this right?
```suggestion
  that lets users request the contents of the API server's `/var/log` directory over HTTP. Accessing
  that endpoint
```

**Aside**: we should perhaps clarify - elsewhere in the docs - that the content you receive depends on which API server you talk to, and what mechanism your cluster uses to distribute client requests between different API server replicas. At least, I think that's true.

Reply:
yeah, it's the content of `/var/log` as seen by the API server handling the request